auth_service/user_management/utils.py

```diff
@@ -0,0 +1,64 @@
+# Copyright 2021 - 2022 Universität Tübingen, DKFZ and EMBL
+# for the German Human Genome-Phenome Archive (GHGA)
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+"""Utilities for the user management package."""
+
+from datetime import datetime, timezone
+
+from pydantic import parse_obj_as
+
+__all__ = ["DateTimeUTC", "now_as_utc"]
+
+
+UTC = timezone.utc
+
+
+class DateTimeUTC(datetime):
+    """A pydantic type for values that should have an UTC timezone.
+
+    This behaves exactly like the normal datetime type, but requires that the value
+    has a timezone and converts the timezone to UTC if necessary.
+    """
+
+    @classmethod
+    def create(cls, *args, **kwargs):
```

Comment:
Does this method name have a special meaning for pydantic? Otherwise I would suggest to name it `construct` to be consistent with hexkit components.

Reply:
Ok, will rename it when moving it to the chassis lib.